src/main/kotlin/reposfinder/main.kt

```diff
@@ -0,0 +1,11 @@
+import reposfinder.config.Config
+import reposfinder.logic.ReposFinder
+
+fun main() {
+    println("Please, input path to .json config file: ")
+    val configPath = readLine()
```

Comment:
It's better to pass the path to config from CLI.
```
./gradlew run configPath
```

Reply:
fixed